components/TextResponse.tsx

```diff
@@ -4,12 +4,16 @@ import { useDebounce } from 'use-lodash-debounce'
 
 import styles from '../styles/TextResponse.module.css'
 
-type Props = {
+export type TextResponseProps = {
   placeholder?: string
   title?: string
   updateCallback?: (update: string) => void
 }
-const TextResponse = ({ placeholder, title, updateCallback }: Props) => {
+const TextResponse = ({
+  placeholder,
+  title,
+  updateCallback
+}: TextResponseProps) => {
```

Comment:
Adding a default placeholder text might be a better approach -- it looks like if we grab keyboard focus the title won't be read! (https://brucelawson.co.uk/2009/the-accessibility-of-html-5-autofocus/)

Reply:
Then the `<textarea>` needs a `<label>`. I raised this because I suppose when people see the text question they will try to type, but if the textarea doesn't have the focus they will have to click into it.

Reply:
Do the same thing for the slider.